ci-operator/step-registry/aro/ocm/register/aro-ocm-register-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "aro/ocm/register/aro-ocm-register-ref.yaml",
+	"owners": {
+		"approvers": [
+			"jianlinliu",
+			"yunjiang29",
+			"mgahagan73",
+			"MayXuQQ"
```

Comment:
I'd suggest adding an OCM team member as an approver for `aro/ocm` folder

Reply:
Got it, I'll check to see who the best person on our team to add is.

Reply:
Please add `xueli181114` and `yasun1`